pkgs/by-name/ca/cargo-mdbook-tabs/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  perl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-mdbook-tab";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "RustForWeb";
+    repo = "mdbook-plugins";
+    rev = version;
+    hash = "sha256-IyIUJH5pbuvDarQf7yvrStMIb5HdimudYF+Tq/+OtvY=";
+  };
+
+  cargoHash = "sha256-/UM85Lhq52MFTjczPRuXENPJOQkjiHLWGPPW/VD9kBQ=";
+
+  nativeBuildInputs = [
+    perl
+  ];
+
+  buildInputs = [
+  ];
+
+  meta = with lib; {
+    description = "mdBook plugin for rendering content in tabs.";
```

```suggestion
    description = "mdBook plugin for rendering content in tabs";
```
The description must not end with a period. (https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
It also states that the description must be capitalized but since this is referring to the name of a program I'm not sure if it applies.